.license_scout.yml

```diff
@@ -254,6 +254,18 @@ exceptions:
       reason: (sd) Shell out only
     - name: core/haproxy
       reason: (sd) Binary use only. Already shipped in chef-server. Exception made by Chef Legal
+    # openjdk11 and it's deps
```

Comment:
These license scout approvals are currently temporary and have not been approved by legal yet. These are to get the thing to build so we can start integration testing with updated ES.  If legal don't approve these we'll probably need to roll our own openjdk11 and exclude alsa and use the bundled freetype library, since the deps that we're allowing here are pulled in for those features that I don't think ES would care about.